cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgsfourleptons.json

```diff
@@ -0,0 +1,92 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan",
+        "orcid": "0000-0002-4485-2972"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "cc",
+        "py",
+        "zip"
+      ],
+      "number_files": 4,
+      "size": 999
+    },
+    "doi": "10.7483/OPENDATA.CMS.TODO",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:999",
+        "size": 999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/TODO/HiggsToFourLeptonsNanoAODOutreachAnalysis-1.0.zip"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12360",
+    "run_period": [
+      "Run2012A",
+      "Run2012B"
+    ],
+    "source_code_repository": {
+      "url": "www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis"
+    },
+    "system_details": {
+      "description": "This code can be used with the CMS Open Data VM environment.",
```

Comment:
With CVMFS it should be straight forward to get any ROOT version, right?

Reply:
Yes, I think so but only for the corresponding OS which may be too old in the CMS open data envs.
But root containers should be available independently as well, I believe

Reply:
Yes, maybe it is best to mention that this code could actually run independently with any root container >6.22 (if it is the case).  Maybe add something like "It is best and more easily used, however, with ...."  Would that work?

Reply:
Indeed, there's no ROOT 6.22 for slc6 on CVMFS. Is there any resource I can link to for the containers? What do you prefer? Otherwise I'd just remove mentioning the VM.

Reply:
If there's no official resource from your side, I'd link to this page: https://hub.docker.com/r/rootproject/root

Reply:
I believe this would be fine.

Reply:
Actually, ROOT has now a really nice install page (https://root.cern/install/). That's probably the best resource you can find out there.

Reply:
Updated!